components/function-controller/config/crd/bases/serverless.kyma-project.io_gitrepositories.yaml

```diff
@@ -54,7 +54,6 @@ spec:
                   description: Type is the type of authentication
                   enum:
                   - basic
-                  - token
```

Comment:
Why did you remove the `token` option? Is it not one of supported options?

Reply:
Custom OAuth implementations differ between popular git providers, I think it's better to add a note to documentation mentioning the possibility of using token auth in a way described by the specific provider.

Reply:
Good point. Thanks!